crates/solidity/outputs/npm/crate/build.rs

```diff
@@ -1,4 +1,11 @@
+// Make sure to always rebuild the cdylib crate when the Rust code changes.
+// See Cargo.toml for more context.
+use slang_solidity as _;
+
+#[cfg(not(feature = "slang_napi_interfaces"))]
+compile_error!("The whole point is to enable slang_napi_interfaces for this cdylib crate!");
```

Comment:
If we ever compile this cdylib without this feature then it's definitely a bug, so I decided to enforce this and this should better show how `slang_napi_interfaces` feature should be used in the context of the entire workspace